tests/e2e/examples-simple-streaming/00-install.yaml

```diff
@@ -0,0 +1,5 @@
+# Start Kafka
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: "pwd && cd ../../.. && make kafka"
```

Comment:
This test if fialing I think we need to change this to 

`  - script: " KAFKA_NAMESPACE=$NAMESPACE make kafka"`